src/coreclr/jit/importer.cpp

```diff
@@ -1616,12 +1616,6 @@ GenTree* Compiler::impAssignStructPtr(GenTree*             destAddr,
     GenTree* asgNode = gtNewAssignNode(dest, src);
     gtBlockOpInit(asgNode, dest, src, false);
 
-    // TODO-1stClassStructs: Clean up the settings of GTF_DONT_CSE on the lhs
```

Comment:
have you tested this change?

Reply:
I am testing it currently.
I haven't seen any issues with it so far
I an willing to remove it from this fix.

Reply:
I think for backport sake we should leave it out.